mvp-spec/partials-updater/src/interpretor.ts

```diff
@@ -81,6 +95,34 @@ async function interpretPartialBeginToken(token: LexerToken) : Promise<string> {
     }
 }
 
+async function transformMermaids(config: PartialConfig) {
+    const lineBreak = getLineBreakSymbol(config.files[0]);
+    const cmdAndPathes = config.files.map(buildMermaidCommand);
+    cmdAndPathes.forEach((val) => {
+        child.execSync(val[0]);
+    });
+    const asyncContents = cmdAndPathes.map((val) => {
```

Comment:
I don't think anything is async here. It seems you have left code that used to be async but it's not anymore.
Also, the `Promise.all()` below is unnecessary.

Reply:
Thank you. I have forgotten part of a refactoring. :-)